src/main/views/MattermostBrowserView.test.js

```diff
@@ -478,4 +481,14 @@ describe('main/views/MattermostBrowserView', () => {
             expect(AppState.updateMentions).toHaveBeenCalledWith(mattermostView.view.id, 0);
         });
     });
+
+    describe('updateMainWindowTitle', () => {
+        MainWindow.get().setTitle('bar');
```

Comment:
This call isn't needed, since you're just calling a mock function.

Reply:
Done.